tests/integration/tier0/destructive/method/main.fmf

```diff
@@ -22,3 +22,6 @@ description: |
 
 /satellite_conversion:
     test: pytest -svv -m satellite_conversion
+
+/pre_registered_system:
+    test: pytest -svv -m test_pre_registered_conversion
```

Comment:
Remove the "test" prefix

Reply:
See https://github.com/oamg/convert2rhel/pull/869#discussion_r1318257741

Reply:
:cry: